boot/zephyr/boards/nrf54l15dk_nrf54l15_cpuapp.conf

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+CONFIG_BOOT_MAX_IMG_SECTORS=512
+
+# Ensure that the qspi driver is disabled by default
+CONFIG_NORDIC_QSPI_NOR=n
+
+# TODO: below are not yet supported and need fixing
+CONFIG_NRF_RTC_TIMER=n
+CONFIG_FPROTECT=n
+CONFIG_BOOT_WATCHDOG_FEED=n
+CONFIG_MULTITHREADING=y
```

Comment:
Why would multithreading be enabled for no reason?

Reply:
soc_flash_nrf_rram uses semaphores which are getting optimized out without multi threading.

Reply:
Why does a flash driver need a semaphore, surely it should be blocking?